support-frontend/app/controllers/DiagnosticsController.scala

```diff
@@ -0,0 +1,39 @@
+package controllers
+
+import actions.CustomActionBuilders
+import play.api.mvc._
+import Results.Ok
+
+class DiagnosticsController(
+  actionRefiners: CustomActionBuilders,
+) {
+
+  val relevantCookies = List(
+    "gu_user_features_expiry",
+    "gu_paying_member",
+    "GU_AF1",
+    "gu_action_required_for",
+    "gu_digital_subscriber",
+    "gu_hide_support_messaging",
+    "gu_recurring_contributor",
+    "gu_one_off_contribution_date",
+    "gu.contributions.recurring.contrib-timestamp.Monthly",
+    "gu.contributions.recurring.contrib-timestamp.Annual",
+    "gu.contributions.contrib-timestamp",
+    "gu_article_count_opt_out",
+    "gu_contributions_reminder_signed_up",
```

Comment:
this list was lifted directly from 
https://github.com/guardian/frontend/blob/126dfcbc1aa961650b7f7ff41ee50a12782bb62e/static/src/javascripts/projects/common/modules/commercial/user-features.js#L9-L37